scipy/ndimage/_ni_docstrings.py

```diff
@@ -69,7 +69,7 @@
 
 _mode_interp_constant_doc = (
 """mode : {'reflect', 'grid-mirror', 'constant', 'grid-constant', 'nearest', \
-           'mirror', 'grid-wrap', 'wrap'}, optional
+'mirror', 'grid-wrap', 'wrap'}, optional
```

Comment:
👍 nice catch.

Reply:
@Yann-P It doesn't look like this renders incorrectly in the documentation, though, right? Sphinx must strip out the extra whitespace?